pkg/util/util.go

```diff
@@ -782,12 +782,11 @@ func SafeGetBool(b *bool) bool {
 
 // IsPortFree checks if the port on localhost is free to use
 func IsPortFree(port int) bool {
-	address := fmt.Sprintf("localhost:%d", port)
+	address := fmt.Sprintf("0.0.0.0:%d", port)
 	listener, err := net.Listen("tcp", address)
 	if err != nil {
 		return false
 	}
-	_ = listener.Addr().(*net.TCPAddr).Port
```

Comment:
What did this do?

Reply:
It can be used to determine the port value on which we are listening (by casting `listener.Addr()` into `*net.TCPAddr`). But the return value is not used at all. And it is useless to me since we are already passing the input `port`. It would have been useful for example if we were letting the kernel bind to a random port and we wanted to get that random port value.